client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts

```diff
@@ -81,20 +81,38 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
           return;
       }
 
-      if(val.length > 2) {
-          this.filteredOptions= [];
-
-          this.placeData.searchAddresses(val).then(function(results){
-              if(results) {
-
-                  results.forEach((result) => {
-                      let address = self.getFullAddress(result);
-                      result.address = address.trim();
-                  });
-
-                  self.filteredOptions = results;
+      if (val.length > 2) {
```

Comment:
A lot of duplicate code here.
Can the code block from here and from the `active-wildfire-map` component move the duplicates into a service or utility function so we only need the code in one place?